README.md

```diff
@@ -367,6 +367,10 @@ shopify.metafield.create({
   - `list([params])`
   - `search(params)`
   - `update(id, params)`
+- giftCardAdjustment [Shopify](https://help.shopify.com/en/api/reference/plus/gift_card_adjustment)
```

Comment:
```suggestion
- giftCardAdjustment
```

For consistency

Reply:
My only issue with this one is that the Gift Card Adjustment API is hidden.

Reply:
I see, do you know if it is hidden on purpose?